collectors/pod.go

```diff
@@ -137,6 +137,18 @@ var (
 		"The limit on memory to be used by a container in bytes.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerResourceRequestsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_requests_gpu",
```

Comment:
@WIZARD-CXY Be consistent with cpu cpu and memory metric name is preferred. 

`kube_pod_container_resource_requests_gpu` -> `kube_pod_container_resource_requests_gpu_cores`
`kube_pod_container_resource_limits_gpu ` -> `kube_pod_container_resource_limits_gpu_cores`